config/testgrids/kubernetes/sig-storage/config.yaml

```diff
@@ -3,7 +3,15 @@ dashboard_groups:
   dashboard_names:
     - sig-storage-kubernetes
     - sig-storage-local-static-provisioner
-    - sig-storage-csi
+    - sig-storage-ci
```

Comment:
Can we have multiple nested dashboard groups? For example, put all of these under a sig-storage-csi group?

Otherwise, would prefer prefixing all of these with "sig-storage-csi"

Reply:
> Can we have multiple nested dashboard groups? For example, put all of these under a sig-storage-csi group?

I'm not sure whether that is supported by testgrid.

> Otherwise, would prefer prefixing all of these with "sig-storage-csi"

So `sig-storage-csi-<something>` with `<something> = ci/other/<repo>`?